recipes/ltd-conveyor/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "ltd-conveyor" %}
+{% set version = "0.7.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ebbc5fd9b86e627366e9c25fe9e8b9c16df4e38c702ee73cb93a9a324bd3ed35
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - ltd = ltdconveyor.cli.main:main
+
+requirements:
+  host:
+    - python >=3.0
```

Comment:
It seems odd to have this as `>=3.0`--any reason not to mirror the `>=3.7` from the `run` section?

Reply:
Thanks for catching that. I've got them synced up now.